src/components/decimal/decimal.spec.js

```diff
@@ -1349,6 +1350,30 @@ describe("Decimal", () => {
       );
     });
   });
+
+  describe("i18n", () => {
+    describe("translation", () => {
+      it("renders properly", () => {
+        mount(
+          <I18nProvider
+            locale={{
+              locale: () => "fr-FR",
+            }}
+          >
+            <Decimal />
+          </I18nProvider>
+        );
+        expect(value()).toBe("0,00");
+        wrapper.setProps({
+          locale: {
+            locale: () => "en-GB",
+          },
+        });
+        wrapper.find('input[data-element="input"]').simulate("blur");
```

Comment:
There is already a `blur()` method you can use.

Reply:
updated, thanks